presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -58,6 +57,7 @@ public final class IcebergSessionProperties
     private static final String MINIMUM_ASSIGNED_SPLIT_WEIGHT = "minimum_assigned_split_weight";
     private static final String NESSIE_REFERENCE_NAME = "nessie_reference_name";
     private static final String NESSIE_REFERENCE_HASH = "nessie_reference_hash";
+    public static final String COMPRESSION_CODEC = "compression_codec";
```

Comment:
```suggestion
    static final String COMPRESSION_CODEC = "compression_codec";
```
nit: default visibility seems enough here.

Reply:
updated